### PR TITLE
PAT-392 Create document upload pages

### PR DESCRIPTION
**Pull request by steverendell**

### CODE REVIEW

File: server/routes/document.js [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+  ['misc', 'Text or other file type'],
+  ['zip', 'Zip archive file'],
+  ['odt', 'Openoffice writer'],
+  ['pptx', 'Powerpoint'],
+  ['xls', 'Excel Spreadsheet'],
+  ['xlsx', 'Excel Spreadsheet'],
+  ['doc', 'Word Document'],
+  ['docx', 'Word Document'],
+])
+
```
Comment (lbennett-moj) [inline server/routes/document.js]: As discussed, questioned whether this was a complete list of allowed file types

File: server/views/pages/document/list.html [hunk 2]
```diff
@@ -21,7 +21,7 @@ <h1 class="govuk-heading-l">Uploaded files</h1>
             {{ govukButton({
                 text: 'Upload new file',
                 classes: 'rightAlignedButton',
-                href: '/document/new/' + nominalData.id,
+                href: '/document/upload/' + nominalData.id,
```
Comment (yasinmustafa) [inline server/views/pages/document/list.html]: As discussed, the URL pattern for anything that is accessed thrpough the nominal home page should be like /nominal/:nominalid/document/upload created a JIRA for thiss as it affects the assessments route as well PAT-401

File: server/views/pages/document/upload.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      })
+    }}
+    {{ offenderBasicDetailsSection(offenderBasic) }}
+
+<div class="govuk-width-container ">
+    <div class="govuk-grid-row govuk-body">
+        <div class="govuk-grid-column-three-quarters">
+            <h1 class="govuk-heading-l">Upload a file</h1>
+            <p>You can upload any type of file to the prisoner's profile using the file selector below.
+                The file size limit is XX MB.</p>
```
Comment (yasinmustafa) [inline server/views/pages/document/upload.html]: Is the XX because this remains unknown?

Comment (steverendell) [reply server/views/pages/document/upload.html]: Indeed - will remove during next code change.

**Pull request merged**